### ISSUE TITLE
Can you provide your dataset like folder style?

### ISSUE THREAD

**Issue by linhduongtuan** [mr7495/COVID-CT-Code#1]
Dear Mohamad,
First, I thank you for your great job.
I am also interested in the topic Covid-19 CT scan now. I am having very promising models to predict Covid-19 CT scans. So I am trying to expand my experiments via external validity. Can you give me your ENTIRE dataset like folder style, e.g. Covid-19 folder containing only images classified from Covid-19 patients, and Non-Covid-19 folder consiting of scans from healthy subjects?
Best regards.
Linh

**Comment by mr7495** [comment]
Dear Linh,
I appreciate your interest in our work.
There are two ZIP files in our dataset, one is "COVID-CTset.zip" that contains the Ct scan images of each patient, which you may be careful about using it because all the CT scans of a COVID-19 infected person does not contain COVID infections.
But in the "Train&Validation.zip" file, you can find the normal and COVID images that you can use to evaluate your model. You can also use [This CSV](https://github.com/mr7495/COVID-CTset/blob/master/all.csv) that reports the class of each image in
"Train&Validation.zip".

**Comment by linhduongtuan** [comment]
Dear Mohamad,
Many thank for your more detailed explanation.
At the beginning, I have looked at Patient_details.csv and have not seen "filename" column to parse data. Now I know how to grasp your dataset. 
So I will use Trans&Validation set for my external validity experiment.
Kindly
Linh

**Comment by mr7495** [comment]
Dear Linh,
I'm so glad that you are using our data to produce strong models and improve COVID-19 detection. 
I hope you will be healthy and fine. 
If you have any other questions, do not hesitate to contact me.
Kind Regards

**Comment by linhduongtuan** [comment]
Dear Mohamad,
I appreciate in your support and kindly words for me.
I have looked around your github repos and known you have a great job about Covid-19 topics.
My paper focused on Covid-19 Chest Xray has been under-reviewing now, and hopefully it should be accepted ASAP (I will release my full codes at that time). Apart codes from the project can be accessed here if you want to take into your consideration (https://github.com/linhduongtuan/Covid-19_Xray_Classifier).
Besides, I am finishing my experiments about Covid-19 CT scans. I have very good results from such datasets from papers: https://arxiv.org/abs/2003.13865; https://www.medrxiv.org/content/10.1101/2020.04.24.20078584v3; and https://www.cell.com/cell/pdf/S0092-8674(20)30551-1.pdf. My results outperform such the original reports, and nearly absolute correction beyon to in-patients. So I am trying to find other datasets to external validity and luckily to find your valuable dataset.
I can show you the confusion matrix that is inferenced on internal test set (I split train:val:test sets ~ 70%:15%:15%, the dataset can be downloaded from here http://ncov-ai.big.ac.cn/download?lang=en) 
![image](https://user-images.githubusercontent.com/22388092/90092272-abf15c80-dd52-11ea-83b5-2ebe475b5841.png)
I wish that your deserve and valuable contribution can help human-being eliminate the SARS-CoV-2 pandemic ASAP.
Best regards
Linh

**Comment by mr7495** [comment]
Dear Linh,
Thank you for sharing your works with me. These are very promising results and I recommend your work.
 I hope you publish these results in a very good journal so that other researchers be able to use them.

Kind Regards
